README.md

```diff
@@ -3,17 +3,37 @@
 [![CircleCI](https://circleci.com/gh/singnet/snet-cli.svg?style=svg)](https://circleci.com/gh/singnet/snet-marketplace-service)
 [![Coverage Status](https://coveralls.io/repos/github/singnet/snet-marketplace-service/badge.svg?branch=master)](https://coveralls.io/github/singnet/snet-marketplace-service?branch=master)
 ## Getting Started  
-#### SingularityNET Marketplace Service  
-The Marketplace Service serves as an index to display curated services on the SingularityNet DApp.
+### SingularityNET Marketplace Services
+The best way to get started is by understanding the marketplace services that powers the decentralized Blockchain marketplace. Here is a high level view on how SingularityNet's services works.
```

Comment:
"The Marketplace Service serves as an index to display curated services on the SingularityNet DApp.Below  is a high level view on how SingularityNet's services works." - lets retain the old line .

Reply:
@anandrgitnirman This repo is a mono repository which contains contract API service which serves as an index to display curated services on the SingularityNet DApp.